cnd/src/main.rs

```diff
@@ -43,12 +43,22 @@ fn main() -> anyhow::Result<()> {
 
     let mut runtime = Runtime::new()?;
 
+    const BITCOIN_BLOCK_CACHE_CAPACITY: usize = 144;
```

Comment:
Since these are consts anyway, should we just inline them inside the constructor?

Reply:
I personally am religious about removing magic numbers (you might have seen me request them on every single review I've done ever :)  Its not a big enough deal for me to ACK stuff but I always ask for them.